metricbeat/module/http/server/_meta/docs.asciidoc

```diff
@@ -2,12 +2,15 @@ This is the server metricset of the module http.
 
 Events sent to the http endpoint will be put by default under the `http.server` prefix. To change this use the `server.paths`
 config options. In the example below every request to `/foo` will be put under `http.foo`.
-
+Also consider using secure settings for the server using TLS/SSL as shown
 ["source","yaml",subs="attributes"]
 ------------------------------------------------------------------------------
 - module: http
   metricsets: ["server"]
   host: "localhost"
+  ssl.certificate_authorities: ["/etc/pki/root/ca.pem"]
```

Comment:
For a setup without mutually authenticated TLS clients this line isn't needed. I think I'd remove it and just show a simple example where the server is listening with HTTPS but not authenticating clients.

Reply:
Agreed, if we not present any value, it will use system default